src/hdmf/spec/spec.py

```diff
@@ -408,40 +411,44 @@ def is_overridden_spec(self, **kwargs):
             return self.is_overridden_attribute(spec)
         return False
 
-    @docval({'name': 'name', 'type': str, 'doc': 'the name of the attribute to the Spec for'})
+    @docval({'name': 'name', 'type': str, 'doc': 'the name of the attribute to check'})
     def is_inherited_attribute(self, **kwargs):
         '''
-        Return True if the attribute was inherited from the parent type, False otherwise
+        Return True if the attribute was inherited from the parent type, False otherwise.
+
+        Raises a ValueError if the spec is not found.
         '''
         name = getargs('name', kwargs)
         if name not in self.__attributes:
             raise ValueError("Attribute '%s' not found" % name)
         return name not in self.__new_attributes
 
-    @docval({'name': 'name', 'type': str, 'doc': 'the name of the attribute to the Spec for'})
+    @docval({'name': 'name', 'type': str, 'doc': 'the name of the attribute to check'})
     def is_overridden_attribute(self, **kwargs):
         '''
-        Return True if the given attribute overrides the specification from the parent, False otherwise
+        Return True if the given attribute overrides the specification from the parent, False otherwise.
+
+        Raises a ValueError if the spec is not found.
         '''
         name = getargs('name', kwargs)
         if name not in self.__attributes:
             raise ValueError("Attribute '%s' not found" % name)
-        return name not in self.__overridden_attributes
+        return name in self.__overridden_attributes
 
     def is_many(self):
         return self.quantity not in (1, ZERO_OR_ONE)
 
     @classmethod
-    def get_data_type_spec(cls, data_type_def):
+    def get_data_type_spec(cls, data_type_def):  # unused
```

Comment:
I assume 'unused' means they are never called anywhere. Are you leaving these here just in case?

Reply:
Yeah, this is a note-to-self. Reading the code, one might think it would be used in every data_type spec, but actually it is not used anywhere.